docs/transform_affine_color_correction.md

```diff
@@ -13,7 +13,7 @@ Euclidean distance between the transformed source color values and the target co
 
  - **Parameters:**
     - rgb_img       - an RGB image with color chips visualized
-    - source_matrix - array of RGB color values (intensity in the range [0-1]) from the image to be corrected where each row is one color reference and the columns are organized as index,R,G,B
+    - source_matrix - array of RGB color values (intensity in the range [0-1]) from the image to be corrected where each row is one color reference and the columns are organized as index,R,G,B (likely created with [plantcv.transform.create_color_card_mask](create_color_card_mask.md) or [plantcv.transform.detect_color_card](transform_detect_color_card.md)).
```

Comment:
`source_matrix` is from `pcv.transform.get_color_matrix`, using a color card mask